src/java/org/apache/cassandra/cql3/terms/Marker.java

```diff
@@ -98,6 +93,12 @@ public Term.Terminal bind(QueryOptions options) throws InvalidRequestException
         }
     }
 
+    @Override
+    public String toString()
+    {
+        return "?";
```

Comment:
I guess we don't have enough information here to know if the marker is named or not?

Reply:
Unfortunately not.